lib/Sema/CSSimplify.cpp

```diff
@@ -587,13 +587,13 @@ getCalleeDeclAndArgs(ConstraintSystem &cs,
   SmallVector<LocatorPathElt, 2> path;
   auto callExpr = callLocator.getLocatorParts(path);
   if (!callExpr) 
-    return std::make_tuple(nullptr, 0, argLabels, hasTrailingClosure);
+    return std::make_tuple(nullptr, false, argLabels, hasTrailingClosure);
```

Comment:
When passing ‘false’ as an argument like this it might be more readable if it said: `/*hasCurriedSelf=*/false`

Reply:
Ah, sure, makes sense!